### PR TITLE
Enable saml2 for login.gov

### PR DESCRIPTION
**Pull request by jbrown-xentity**
Install the necessary components for login.gov pysaml configuration, but leave off usage (to be utilized/configured by environment).

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -75,8 +75,10 @@ Werkzeug==0.15.3         # via flask
 zope.interface==4.3.2
 
 # ckanext-saml2 dependencies
+defusedxml==0.6.0
 M2Crypto
--e git+https://github.com/GSA/pysaml2.git@max#egg=pysaml2
+python-memcached==1.48
+-e git+https://github.com/GSA/pysaml2.git@feature/ci-login-gov#egg=pysaml2
```
Comment (adborden) [inline requirements.txt:81]: This doesn't look right, should be datagov/v4.9.0

File: requirements.txt [hunk 2]
```diff
@@ -2,7 +2,7 @@
 -e git+https://github.com/GSA/ckanext-googleanalyticsbasic#egg=ckanext-googleanalyticsbasic
 -e git+https://github.com/GSA/USMetadata.git@ckan-2-8#egg=ckanext-usmetadata
 -e git+https://github.com/GSA/ckanext-datajson.git@master#egg=ckanext-datajson
--e git+https://github.com/GSA/ckanext-saml2.git@max#egg=ckanext-saml2
+-e git+https://github.com/GSA/ckanext-saml2.git@datagov-0.4.0-config#egg=ckanext-saml2
```
Comment (adborden) [inline requirements.txt:5]: Should be datagov/v0.4.0

**Review by adborden — COMMENTED** [review]
Versions don't look right, they should be main branches, not feature branches or tags.

**Comment by adborden** [comment]
I've asked the ckan-multi folks to address this https://github.com/GSA/datagov-ckan-multi/issues/348#issuecomment-740242511

**Comment by jbrown-xentity** [comment]
Branches created, and fixed locally. Latest commit pulled. Needs to be merged before https://github.com/GSA/datagov-deploy/pull/2519 (so that the datagov-deploy PR can utilize the normal branch)...

**Review by FuhuXia — APPROVED** [review]

File: requirements.txt [hunk 3]
```diff
@@ -75,8 +75,10 @@ Werkzeug==0.15.3         # via flask
 zope.interface==4.3.2
 
 # ckanext-saml2 dependencies
+defusedxml==0.6.0
```
Comment (adborden) [inline requirements.txt:78]: I'm not seeing this as a dependency for ckanext-saml2... are you sure?

**Pull request merged**